internal/api/attestationconfigapi/cli/azure.go

```diff
@@ -27,6 +27,8 @@ func deleteAzure(ctx context.Context, client *attestationconfigapi.Client, cfg d
 	return client.DeleteSEVSNPVersion(ctx, variant.AzureSEVSNP{}, cfg.version)
 }
 
+func ptr[A any](a A) *A { return &a }
+
```

Comment:
This is our canonical ptr implementation: 
```go
func toPtr[T any](v T) *T {
	return &v
}
```
Please also move this to the bottom of the file as it is almost always the least important thing in a file.

Reply:
Thanks for the suggestion, adopted in the other PR.